### PR TITLE
feat(nms): Remove Material-table and use material-table/core instead

### PR DESCRIPTION
**Pull request by HannaFar**
Signed-off-by: HannaFar <hannafarag159@gmail.com>

## Summary

Remove Material-table and use material-table/core instead

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by thmsschmitt** [comment]
We are using `filefy` in SubscriberTable, but we didn't include it in our dependencies and now it got removed see https://github.com/magma/magma/runs/5835150695?check_suite_focus=true

File: nms/packages/magmalte/app/components/ActionTable.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ import FilterList from '@material-ui/icons/FilterList';
 import FirstPage from '@material-ui/icons/FirstPage';
 import FormControl from '@material-ui/core/FormControl';
 import LastPage from '@material-ui/icons/LastPage';
-import MaterialTable from 'material-table';
+import MaterialTable from '@material-table/core';
```
Comment (thmsschmitt) [inline nms/packages/magmalte/app/components/ActionTable.js:33]: Unfortunately we have to drop the `strict-local` above

Comment (HannaFar) [reply nms/packages/magmalte/app/components/ActionTable.js:33]: thanks!

**Review by andreilee — APPROVED** [review]
Fix the yarn.lock issue before merging, please

**Pull request merged**